tests/test_config.py

```diff
@@ -241,6 +241,11 @@ def test_style_pep257(self, test_file: str) -> None:
         run = _Run([test_file, "--style", "pep257"])
         assert run.config.style == ["pep257"]
 
+    def test_style_numpydoc_only(self, test_file: str) -> None:
```

Comment:
👍

Reply:
It would probably be useful to check that the formatter activation values are getting set properly.  Should that go in these tests or in separate tests?

Reply:
I think we already have tests for those. But if you can come up with new ones they should probably be their own tests/functions.

Reply:
With the new `--style` options?  Admittedly that's probably less of an issue with #145 than my original implementation.

Reply:
Yeah, it should no longer depend on the `style` options`. If `FormatterA` has style `["numpydoc"]` it can be "turned on" but still be disabled when `style` does not include `numpydoc`.